samples/app/src/androidTest/java/com/sourcepoint/app/v6/MainActivityKotlinNativeMessage.kt

```diff
@@ -58,7 +58,7 @@ class MainActivityNativeMessTest {
         +(CampaignType.CCPA)
     }
 
-    @Test
+//    @Test
```

Comment:
Did you comment this test on purpose?

Reply:
yes, we need to work on the stability of tests since the workflow changed